bin/rabbitstressor.js

```diff
@@ -16,10 +16,27 @@ class RabbitStressor {
     this.channel = await this.connection.createChannel();
   }
 
-  sendMessages(queueName, messages) {
+  sendMessages(queueName, messages, delayBetweenMessages) {
     assert(messages instanceof Array);
     this.channel.assertQueue(queueName, {durable: false});
-    messages.forEach(message => this.channel.sendToQueue(queueName, new Buffer(message)));
+
+    return new Promise(resolve => {
+      let messagesDeepCopy = JSON.parse(JSON.stringify(messages));
+      this._recursiveTimeout(queueName, messagesDeepCopy, delayBetweenMessages, resolve);
+    });
+  }
+
+  _recursiveTimeout(queueName, messageQueue, delayBetweenMessages, resolve) {
```

Comment:
Yeah, this works the same as `setInterval`. I added a patch using `setInterval` which does the same thing as the code snippet above (except that it is not recursive!). Let me know which one you prefer.